app/javascript/src/component_expander.js

```diff
@@ -160,9 +160,16 @@ class Expander {
       'aria-controls': this.#id
     });
 
-    $activator.on("click", () => {
-      this.toggle();
-    });
+
+    if(this.#isInput($activator)) {
+      $activator.on("change", () => {
+        this.toggle();
+      });
+    } else {
+      $activator.on("click", () => {
+        this.toggle();
+      });
+    }
```

Comment:
Think `isInput()` is used a couple of times so I'm wondering why we don't set the event type inside the `createActivator()` method, or whatever suitable place. Then we can reduce lines with like: 
```
$activator.on(activator_event_type, () => {
        this.toggle();
      });
```
Might even be able to ditch the curly braces but not 100% sure on syntax.
Readability over streamlined is probably more important than clipping two characters, anyway :-)

Reply:
Personal preference maybe, but I can't stand arrow functions without curly braces! 😀